setup.cfg

```diff
@@ -27,6 +27,7 @@ install_requires =
     setuptools
     types-setuptools
     requests>=2.31.0
+    parsys-requests-unixsocket==0.3.1
```

Comment:
Any reason for the strict version pinning?

Reply:
Didn't really think about it when adding it, should it be `>=0.3.1`?